packages/ember-core/src/components/popover.gts

```diff
@@ -159,12 +161,17 @@ export default class Popover extends Component<PopoverSignature> {
     this.arrow = popover;
   };
 
-  show = async (evtOrInput: Event | HTMLInputElement) => {
+  show = restartableTask(async (evtOrInput: Event | HTMLInputElement) => {
```

Comment:
I'm not sure I like this being a task that has to be `perform`ed. Instead, can you keep `show` as an async function and add a new task to handle the timeout? That way, users of the popover components don't need to know that it's a task.